cypress/integration/thread/action_bar_spec.js

```diff
@@ -178,7 +178,7 @@ describe('action bar renders', () => {
       triggerThreadDelete();
     });
 
-    it('should edit the thread', () => {
+    it.only('should edit the thread', () => {
```

Comment:
That's one way to fix e2e tests... :trollface:

Reply:
Whoooopppsss